spec/stylers/ui_label_styler.rb

```diff
@@ -21,6 +21,11 @@ def ui_label_centered(st)
     st.text_alignment = :centered
   end
 
+  def ui_label_text_align(st)
+    ui_label_kitchen_sink(st)
+    st.text_align = :left
```

Comment:
I should switch to right.   I was using left as an alternative to kitchen sink which is making it :center.   But I'd like the test to be atomic, so good call.   I'll switch it to :right.